api/build.gradle

```diff
@@ -157,7 +161,7 @@ dependencies {
 
 test {
   testLogging {
-    events "passed", "skipped", "failed", "standardOut", "standardError"
```

Comment:
Where do stdout and stderr go with this?

Reply:
I'm not sure, but "anywhere besides my terminal window" is the only answer I care about.

Reply:
So... how do you debug failing tests?

Reply:
Also, does this influence what happens when we run tests in circle?

We definitely need to make sure we're capturing output somewhere accessible there...